src/encoded/tests/test_validation_errors.py

```diff
@@ -7,11 +7,17 @@
 
 notice_pytest_fixtures(app_settings, workbook)
 
-pytestmark = [
-    pytest.mark.working, pytest.mark.indexing,
-    # pytest.mark.flaky(rerun_filter=delay_rerun),
-]
-
+# skip all tests tests since validation_errors removed in search.py
+# https://stackoverflow.com/a/42512169
+if pytest.__version__ < "3.0.0":
+  pytest.skip()
+else:
+  pytestmark = pytest.mark.skip
```

Comment:
Are you running Python < 3.6 anywhere? I don't think is needed. We don't officially support <3.6 so I am hesitant to accept this change.

Reply:
@willronchetti Well, you are right; let me recheck it. (At first, I had added `pytest.mark.skip` decorator to tests in test_validation_errors.py to skip all tests instead of deleting the file, but both local and Github Actions failed. At the end of some trials and errors, those lines above help me skip the tests.)